src/sgx/signing/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+pwd=$PWD
+cd "$(dirname "$0")"
+
+echo --- Build
+(
+  set -x
+  openssl genrsa -out "$pwd"/temp_priv_key.pem -3 3072
```

Comment:
looks like you'll want a temporary output file for this and stuff like this in this project to keep git quiet